### ISSUE TITLE
targetSdkVersion 28 using flutter 1.0.0

### ISSUE THREAD

**Issue by kbrmimbyl** [flutter/flutter#38034]
Google Play wants all apps to target 28 starting Nov 1st 2019. I'm running Flutter 1.0.0 and my app is very complex. Can't compile if I upgrade flutter. So -- how do I properly use targetSdkVersion 28 using Flutter 1.0.0 itself?

**Comment by jld3103** [comment]
I think the only option is to upgrade.
Why can't you compile then?

**Comment by kbrmimbyl** [comment]
If I upgrade, lots of places in the code report errors.

Okay, to clarify a bit more, I was able to simply change targetSdkVersion
to 28 in the app's build.gradle and get apk's out. But is that all I need
to do?

**Comment by jld3103** [comment]
Basically yes, but can you clarify which errors you get? Without them your report is pretty much useless :)

**Comment by kbrmimbyl** [comment]
I'm sorry, I can't even begin to list those errors. All plugins fail, even my own code fails because the latest stable Flutter logs syntax errors which aren't errors in 1.0.0.

**Comment by jld3103** [comment]
oh...

**Comment by kbrmimbyl** [comment]
Here's my experience trying to bump up targetSdkVersion to 28 using flutter 1.0.0. Everything actually went smooth but on Android 9 devices the keyboard wouldn't appear when it needed to (as attested by https://github.com/flutter/flutter/issues/19644).

So I had no option but to upgrade Flutter to the latest version. I did that using @passsy's flutter_wrapper. I had to upgrade 4-5 plugins including Cached Network Image. Also, the keyword "Future", wherever I had used it, needed me to import dart:async. After all this it compiled with targetSdkVersion 28 and compileSdkVersion 28 in app/build.gradle.

Things look fine for now. It was less scarier than I thought.
